pkg/imgpkg/cmd/file_flags.go

```diff
@@ -10,11 +10,14 @@ import (
 type FileFlags struct {
 	Files []string
 
-	FileExcludeDefaults []string
+	ExcludedFileBasenames []string
 }
 
 func (s *FileFlags) Set(cmd *cobra.Command) {
 	cmd.Flags().StringSliceVarP(&s.Files, "file", "f", nil, "Set file (format: /tmp/foo, -) (can be specified multiple times)")
 
-	cmd.Flags().StringSliceVar(&s.FileExcludeDefaults, "file-exclude-defaults", []string{".git"}, "Excluded file paths by default (can be specified multiple times)")
+	cmd.Flags().StringSliceVar(&s.ExcludedFileBasenames, "file-exclude-defaults", []string{".git"}, "Excluded file paths by default (can be specified multiple times)")
+	cmd.Flags().MarkDeprecated("file-exclude-defaults", "use '--file-base-exclude' instead")
+
+	cmd.Flags().StringSliceVar(&s.ExcludedFileBasenames, "file-base-exclude", []string{".git"}, "Exclude all files whose base name matches (format: bar.yaml) (can be specified multiple times)")
```

Comment:
it seems that https://github.com/vmware-tanzu/carvel-imgpkg/blob/87884a5de02e2d4cb0e409cc0e5bf64f88129bab/pkg/imgpkg/image/tar_image.go#L73 is actually using a relative path to the "root directory". so if .git was within a nested dir, exclude would have to be `nested-dir/.git`.

`--file-exclusion` might be a better name imho (since file in our uses implies a path).